usaspending_api/api_contracts/contracts/award/idv/Activity.md

```diff
@@ -0,0 +1,102 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# IDV Activity [/api/v2/awards/idvs/activity/]
+
+This endpoint is used to power the IDV (Indefinite Delivery Vehicle) Activity visualization on IDV Summary Pages. It returns information about child awards and grandchild awards for a given IDV.
+
+## POST
+
++ Request (application/json)
+    + Attributes (object)
+        + `award_id`: `CONT_AW_4730_-NONE-_GS23F0170L_-NONE-` (required, string)
+            Either a "generated" natural award id (string) or a database surrogate award id (number).  Generated award identifiers are preferred as they are effectively permanent.  Surrogate award ids are retained for backward compatibility but are deprecated.
+        + limit: 10 (optional, number)
+            The number of results to include per page.
+            + Default: 10
+        + page: 1 (optional, number)
+            The page of results to return based on the limit.
+            + Default: 1
+
++ Response 200 (application/json)
+    + Attributes
+        + results (array[ChildAward], fixed-type)
+            Results are sorted by obligated amount in descending order.
+        + page_metadata (PageMetaData)
+
+    * Body
+            {
+                "results": [
+                    {
```

Comment:
We've been shifting away from specific data in API contract results. Can we make a new datatype with these objects?

Reply:
@BrianZito Since we were removing specific values in the response objects, a few devs started to include the Body as an example response.